cross_list_courses/templates/cross_list_courses/index.html

```diff
@@ -0,0 +1,59 @@
+{% load collections %}
+{% load django_helpers %}
+{% load static %}
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport"
+          content="width=device-width, initial-scale=1, user-scalable=no">
+    <link href="/static/favicon.ico" rel="icon" type="image/x-icon">
+    <link rel="stylesheet" type="text/css"
```

Comment:
Any reason these `link` elements are formatted differently? Consistency would be helpful for readability.